.github/workflows/validation_trigger.yml

```diff
@@ -18,13 +18,14 @@ jobs:
   # Validate poliastro's new features
   validation:
     runs-on: ubuntu-latest
-    environment: validation-env
```

Comment:
Shouldn't we keep this? Perhaps if we remove the line, the env vars are not properly read